optuna/storages/_in_memory.py

```diff
@@ -3,6 +3,12 @@
 import threading
 import uuid
 
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
```

Comment:
A nit but you can place the `typing` imports with the other std imports, without separating them with an empty line. Ditto for the other files in this PR 🙂 .

Reply:
Thank you! I fixed them.